### PR TITLE
[WIP] Add python3 template for arm

### LINKED ISSUE #13: Python 2/3: Return value instead of using `print`
Please can we update the Python 2 / 3 templates so that they use `return` instead of `print`.

This also needs to be tested.

### PR DESCRIPTION
**Pull request by viveksyngh**
Signed-off-by: Vivek Singh <vivekkmr45@yahoo.in>

fixes https://github.com/openfaas/templates/issues/13

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Which issue(s) this PR fixes 

Fixes #13

## How Has This Been Tested?

Tested on Docker for Mac
`python-armhf` is tested on `scaleway` server with `armv7`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by viveksyngh** [comment]
No.. We don't. Should we add that?

**Comment by alexellis** [comment]
I think so. It should even be the same base image

**Comment by viveksyngh** [comment]
I will add that, Should I update this PR or raise a different pull request from that?

**Comment by alexellis** [comment]
Let's update this one please

**Comment by viveksyngh** [comment]
@alexellis I have updated this PR with new template `python3` template for `armhf`.

File: template/python3-armhf/template.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language: python3-arhmf
```
Comment (alexellis) [inline template/python3-armhf/template.yml]: Small typo here - should be `armhf`. Please can you amend the commit and push again?

File: template/python3-armhf/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
+FROM armhf/python:3.6-alpine
```
Comment (alexellis) [inline template/python3-armhf/Dockerfile]: I'm not sure that the prefix is needed of `armhf` anymore since the mutli-arch builds are now official.

**Comment by alexellis** [comment]
LGTM. I'll merge.

File: template/python3-armhf/Dockerfile (hunk 2 above)
Comment (alexellis) [reply template/python3-armhf/Dockerfile]: (hmm.. `python:3.6-alpine` doesn't exist, but `armhf/python:3.6-alpine` does.. we should check the library via @tiannon to see where the Python 3 / Alpine image is for ARM )

**Pull request merged**